log_config.go

```diff
@@ -134,6 +137,8 @@ func (c LogConfig) getFormatter() log.Formatter {
 		return &log.JSONFormatter{}
 	case Logstash:
 		return getLogstashFormatter(c.FormatSettings)
+	case Stackdriver:
+		return &stackdriver.Formatter{}
```

Comment:
I think it is better to use `stackdriver.NewFormatter()` here. Also it makes sense to add optional `service` and `version` format settings and pass them to builder if set.

Reply:
@vgarvardt Sure. Makes sense. I'll implement the same pattern as for the current getLogstashFormatter().